virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -118,6 +118,10 @@ The SVVP Certification applies to:
 // CNV-29848: Release: Mechanism to manage DP/TP features in the UI
 * Cluster administrators can enable or disable link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features on the link:https://docs.openshift.com/container-platform/4.14/virt/getting_started/virt-web-console-overview.html#overview-settings-cluster_virt-web-console-overview[*Settings*] tab on the *Virtualization* -> *Overview* page.
 
+// CNV-28720
+// pending merge of bootable volumes PR
+* The *DataSources* page and the *Bootable volumes* page have been merged into the xref:../../virt/getting_started/virt-web-console-overview.adoc#bootablevolumes-page_virt-web-console-overview[*Bootable volumes* page] so that you can manage these similar resources in a single location.
```

Comment:
Text is fine as is, but consider if it might read better doing:

```suggestion
* The *DataSources* and *Bootable volumes* pages have been merged into the xref:../../virt/getting_started/virt-web-console-overview.adoc#bootablevolumes-page_virt-web-console-overview[*Bootable volumes* page] so that you can manage these similar resources in a single location.
```